github/azure-self-hosted-runners/README.md

```diff
@@ -0,0 +1,64 @@
+# Self-hosted Github Runners with Garm
+
+Those instructions deploy Garm for self-hosted github runners on azure (c)VMs. Garm listens to webhooks from github and spawns VMs on demand.
+
+## Build
+
+```bash
+docker build -t garm-azure .
+```
+
+## Service
+
+The application is deployed as an [ACI](https://azure.microsoft.com/en-us/products/container-instances) Container Group. It is bootstrapped with an init container which creates an admin user, an initial repository registration, and a pool for the repository. It keeps database state on a Storage Account-backed volume. The app will expose an HTTP/S ports for github webhooks.
+
+### Configuration
+
+The configuration is currently inlined in the `/init.sh` file. Not that you need to purge the state on the storage account on changes, otherwise initialization will be skipped.
```

Comment:
The second sentence feels disconnected, do you mean we should not purge or we should purge?